cost/kubecost/cluster/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.3.3
+
+- Added support for multiple currencies
```

Comment:
This should be more specific. I would change it to something like:

Currency values in incident table are automatically converted to the local currency of the Flexera organization using the most recent exchange rate.

Reply:
done!